packages/ckeditor5-list/src/listproperties/listpropertiesediting.ts

```diff
@@ -256,9 +257,10 @@ export interface AttributeStrategy {
  */
 function createAttributeStrategies( enabledProperties: ListPropertiesConfig ) {
 	const strategies: Array<AttributeStrategy> = [];
+	const normalizedConfig = getNormalizedConfig( enabledProperties );
 
 	if ( enabledProperties.styles ) {
-		const useAttribute = typeof enabledProperties.styles == 'object' && enabledProperties.styles.useAttribute;
+		const useAttribute = typeof enabledProperties.styles == 'object' && normalizedConfig.useAttribute;
```

Comment:
```suggestion
		const useAttribute = normalizedConfig.styles.useAttribute;
```
Keeping the `styles` scoping of normalized config would be better.